app/controllers/home_controller.rb

```diff
@@ -0,0 +1,7 @@
+class HomeController < ApplicationController
+
+	def index
+	
+		
+	end
```

Comment:
A shorthand way of writing this method would be ```def index; end```. This makes it easier for us to read.